spec/formatting.md

```diff
@@ -363,7 +363,7 @@ These are divided into the following categories:
     uses a `:plural` selector function which requires its input to be numeric:
 
     ```
-    match {|horse| :plural}
+    match {horse :plural}
     when 1 {The value is one.}
```

Comment:
This is the part that makes me a bit skeptical. I see `{horse` in one line and `{The` in the next; they look the same but are completely different syntax. Prefixes like `$`, `:`, and `|` make it easier for me to visually recognize placeholders.

Do you think we could keep the quoted literals in the examples in this file, as a stylistic choice?